identity/app/views/profile/accountDetailsForm.scala.html

```diff
@@ -190,6 +194,27 @@ <h2 class="form__heading">Billing address</h2>
         </fieldset>
     }
 
+    <fieldset class="fieldset">
+        <div class="fieldset__heading">
+            <h2 class="form__heading">Profiling</h2>
+        </div>
+        <div class="fieldset__fields">
+            <ul class="u-unstyled">
+                @accountDetailsForm.error("statusFields.allowThirdPartyProfiling").map { error =>
+                    <div class="form__error">@error.message</div>
+                }
+
+                <label class="label">In addition to the data that you provide to us, we may also match profiling data from third parties with your registration details.</label>
+
+                @checkboxField(
+                    Checkbox(
+                        accountDetailsForm("allowThirdPartyProfiling"),
+                        ('_label, "Allow matching with third party data"), Symbol("data-test-id") -> "profiling")
```

Comment:
I'm not sure I know what `Symbol("data-test-id" -> "profiling")` is for

Reply:
This pattern is used in the rest of the form.